Integrations/MACVendors/MACVendors_description.md

```diff
@@ -0,0 +1,37 @@
+# Integration Instructions
```

Comment:
this should be in `README.md` only
here you can keep the overview if you would like